pom.xml

```diff
@@ -376,8 +376,8 @@
         <version>${maven-plugin-tools.version}</version>
         <configuration>
           <goalPrefix>hpi</goalPrefix>
-          <skipErrorNoDescriptorsFound>true</skipErrorNoDescriptorsFound>
-          <!-- see MNG-5346 -->
+          <extractors>java-annotations</extractors>
```

Comment:
we are only using java annotations, not javadoc, so no need to look for javadoc comments, or bsh or the other deprecated types.